tmtools/helpers.py

```diff
@@ -0,0 +1,77 @@
+"""Helper tools to work with TM-Align."""
+
+from Bio.PDB.Structure import Structure
+import numpy as np
+
+
+def apply_tmalign_transformation(original_structure, tmalign_result):
+    """
+    Transform structure based on TM-Align result.
+
+    This function creates a copy of the input structure and applies the rotation
+    matrix and translation vector from a TM-align result to all atoms in the copied
+    structure.
+
+    Parameters
+    ----------
+    original_structure : Bio.PDB.Structure.Structure
+        The original structure to transform.
+    tmalign_result : object
```

Comment:
```suggestion
    tmalign_result : TM_result
```

Reply:
Changing this change to the two "transformation" functions.